Makefile

```diff
@@ -25,8 +25,12 @@ GO_GCFLAGS := $(shell if $(GO) version|grep -q gccgo; then echo "-gccgoflags"; e
 NPROCS := $(shell nproc)
 export GO_BUILD=$(GO) build
 export GO_TEST=$(GO) test -parallel=$(NPROCS)
-#RACEFLAGS := $(shell $(GO_TEST) -race ./pkg/dummy > /dev/null 2>&1 && echo -race)
-RACEFLAGS :=
+# If RACEFLAGS is defined then set appropriate RACEFLAGS automatically for `go test`
+ifdef RACEFLAGS
+	RACEFLAGS := $(shell $(GO_TEST) -race ./pkg/dummy > /dev/null 2>&1 && echo -race)
+else
+	RACEFLAGS :=
+endif
```

Comment:
I think this should just be
`RACEFLAGS ?= `

https://www.gnu.org/software/make/manual/html_node/Setting.html